File: src/main/java/matteot92/prenotauncambiolook/model/entities/Ordine.java
package matteot92.prenotauncambiolook.model.entities;

import java.time.LocalDate;
import java.time.LocalTime;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name = "ordini")
@Getter
@Setter
@ToString
@NoArgsConstructor
@JsonPropertyOrder({ "id", "data", "orario", "quantita", "servizio", "utente", "isPagato"})
public class Ordine {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "id_ordine")
	private Long id;
	@Column(nullable=false)
	private LocalDate data;
	@Column(nullable=false)
	private LocalTime orario;
	@Column(nullable=false)
	private Integer quantita;
	@Column(name="id_servizio")
	private Long servizio;
	@Column(name="id_utente")
	private Long utente;
	private Boolean isPagato;
	
}

File: src/main/resources/frontend/src/app/models/ordine.ts
export class Ordine {

  id: number;
  data: Date;
  orario: Date;
  quantita: number;
  servizio: number;
  utente: number;
  isPagato: boolean;

  constructor(id: number, data: Date, orario: Date, quantita: number, servizio: number, utente: number, isPagato: boolean) {
    this.id = id;
    this.data = data;
    this.orario = orario;
    this.quantita = quantita;
    this.servizio = servizio;
    this.utente = utente;
    this.isPagato = isPagato;
  }

}
